components/locid_transform/src/fallback/algorithms.rs

```diff
@@ -484,4 +547,39 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn test_fallback_transliteration() {
+        let cases: &'static [(&'static str, &'static [&'static str])] = &[
+            ("en-US", &["en-US", "en", "und-Latn"]),
+            (
+                "az-Arab-IR",
+                &["az-Arab-IR", "az-Arab", "az-IR", "az", "und-Arab"],
+            ),
+            ("ru-RU", &["ru-RU", "ru", "und-Cyrl"]),
```

Comment:
please add `expected_transliteration_chain` to all existing test cases, to better compare behaviour and have more coverage. You can add more test cases as well.